src/types/data.ts

```diff
@@ -104,6 +104,7 @@ interface Notification {
 	focusTitle?: string;
 	kaid?: string;
 	kind: string;
+	name: string;
```

Comment:
This key isn't always available for every notification, so let's use `name?:` instead here. Same for `translatedRequirements` below. After that, I'll merge.

Reply:
Okay, done